### ISSUE TITLE
map.addControl error

### ISSUE THREAD

**Issue by hishamkaram** [jonataswalker/ol-geocoder#140]
code:
```
componentDidMount() {
        const { childrenProps } = this.props
        childrenProps.map.setTarget(this.mapDiv)
        var geocoder = new Geocoder( 'nominatim', {
            provider: 'photon',
            targetType: 'text-input',
            lang: 'en',
            placeholder: 'Search for ...',
            limit: 5,
            keepOpen: false
        } )
        childrenProps.map.addControl( geocoder )
    }
```
error:
```
ol.js?aa24:142 Uncaught TypeError: (intermediate value)(intermediate value)(intermediate value).appendChild is not a function
    at n.qg.setMap (ol.js?aa24:142)
    at G.eval (ol.js?1912:282)
    at Yc.b (ol.js?1912:42)
    at Yc.Qc.b (ol.js?1912:45)
    at Yc.k.He (ol.js?1912:49)
    at Yc.k.push (ol.js?1912:49)
    at G.k.kk (ol.js?1912:283)
    at ContentGrid.componentDidMount (ContentGrid.jsx?195d:87)
    at commitLifeCycles (react-dom.development.js?cada:8798)
    at commitAllLifeCycles (react-dom.development.js?cada:9967)
```
openlayers version: `openlayers@4.2.0`
react version: `react@16.1.1`

**Comment by hishamkaram** [comment]
fixed with `openlayers@4.4.2` @jonataswalker

**Comment by jonataswalker** [comment]
Ah, fine.
